store/sqlstore/team_store.go

```diff
@@ -384,16 +384,13 @@ func (s SqlTeamStore) GetAllPage(offset int, limit int) store.StoreChannel {
 	})
 }
 
-func (s SqlTeamStore) GetTeamsByUserId(userId string) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		var data []*model.Team
-		if _, err := s.GetReplica().Select(&data, "SELECT Teams.* FROM Teams, TeamMembers WHERE TeamMembers.TeamId = Teams.Id AND TeamMembers.UserId = :UserId AND TeamMembers.DeleteAt = 0 AND Teams.DeleteAt = 0", map[string]interface{}{"UserId": userId}); err != nil {
-			result.Err = model.NewAppError("SqlTeamStore.GetTeamsByUserId", "store.sql_team.get_all.app_error", nil, err.Error(), http.StatusInternalServerError)
-			return
-		}
+func (s SqlTeamStore) GetTeamsByUserId(userId string) ([]*model.Team, *model.AppError) {
+	var data []*model.Team
```

Comment:
nit: Up to you, but probably makes sense to use a more meaningful variable name like `teams` (I know it was `data` before)